tableaudocumentapi/connection.py

```diff
@@ -26,6 +26,11 @@ def __init__(self, connxml):
         self._dbname = connxml.get('dbname')
         self._server = connxml.get('server')
         self._username = connxml.get('username')
+        self._authentication = connxml.get('authentication')
+        self._class = connxml.get('class')
+
+    def __repr__(self):
```

Comment:
Hahaha. Okay ... then Connection makes sense from the object sense. I guess we should figure out if we want it to look like the xml or the object (which might have more information) and then be clear. Perhaps your initial pass was better to make it clear it is not the raw xml and we could have a different way to dump that. I am cool with that